docs/02-guides/display-wysiwyg-content.mdx

```diff
@@ -0,0 +1,344 @@
+
+We are solving this by creating a new Wysiwyg Type for each service. In the
+previous section, we've mentioned the type `MagentoWysiwyg`. It uses the exact
+same mechanisms as the one we will describe below.
+
+If you want to customize `MagentoWysiwyg` instead, please refer to
+[Wysiwyg Plateform section](./customize-wysiwyg-platform.mdx#magentowysiwyg).
+
+#### Register your widget type at the GraphQL level
+
+1.  Add `Front-Commerce/Wysiwyg` as a dependency of your GraphQL module (for
+    instance at `extensions/acme-extension/modules/wysiwyg/`)
+1.  Define a new GraphQL type that should implement the `Wysiwyg` interface
+    ```ts title="extensions/acme-extension/modules/wysiwyg/index.ts"
+    typeDefs: /* GraphQL */ `
+      type CustomWysiwyg implements Wysiwyg {
+        raw: String
+        childNodes: JSON
+        data: [WysiwygNodeData]
+      }
+    `,
+    ```
+1.  Register this new type as a Wysiwyg type in your GraphQL module
+    ```ts title="extensions/acme-extension/modules/wysiwyg/runtime.ts"
+    contextEnhancer: ({ loaders }) => {
+      loaders.Wysiwyg.registerWysiwygType(
+        // the name of the GraphQL type you have just created
+        "CustomWysiwyg",
+        // The list of shortcodes that are parsed and replaced in your wysiwyg content
+        [
+          {
+            // regex to identify the shortcode within your HTML code
+            regex: /\{\{media url="(.*?)"\}\}/gi,
+            // How to replace the given shortcode
+            replacement: (match, url) => `/media/${url}`,
+          },
+        ]
+      );
+    };
+    ```
+
+#### Map this new type to a specific React component that will apply the needed transformations before displaying the content
+
+1.  Duplicate the
+    [`DefaultWysiwyg.js`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/3.4.0/packages/theme-chocolatine/theme/modules/Wysiwyg/DefaultWysiwyg.js)
+    component to `theme/modules/Wysiwyg/CustomWysiwyg`.
+1.  Override `theme/modules/Wysiwyg/appWysiwygComponents.js` and map your new
+    `CustomWysiwyg` typename (the one used in your GraphQL schema) to
+    `theme/modules/Wysiwyg/CustomWysiwyg` (the file that tells React how to
+    display the content)
+
+    ```ts title="theme/modules/Wysiwyg/appWysiwygComponents.js"
+    import CustomWysiwyg from "theme/modules/Wysiwyg/CustomWysiwyg";
+
+    export default {
+      CustomWysiwyg,
+    };
+    ```
+
+That's it, you have a new `CustomWysiwyg` type. However, the components rendered
+are still simple HTML tags. If you want to add interactivity, you will need to
+refer to the `theme/modules/Wysiwyg/CustomWysiwyg` you have created.
+
+In this file you will notice that there is a `defaultComponentsMap` constant.
+This object is used to reference how to render each html tag in your Wysiwyg
+component. We call these **transforms**: they transform an html tag into a React
+component. This is what will allow you to finely tune how your Wysiwyg is
```

Comment:
Some times you use `WYSIWYG` other times `Wysiwyg`, try to update everything to use the same casing.

I prefer `Wysiwyg` as it reads easier, but `WYSIWYG` is probably more correct, as it's an acronym

Reply:
Good point, I'll make a pass and rename everything to the acronym version.